packages/node_modules/@ciscospark/widget-base/src/browser-globals.js

```diff
@@ -0,0 +1,149 @@
+import uuid from 'uuid';
+import Events from 'ampersand-events';
+
+export function setupWidgets() {
+  if (!window.ciscospark.widget || typeof window.ciscospark.widget !== `function`) {
+    window.ciscospark.widget = function widget(el) {
```

Comment:
Each time `setupWidgets()` gets called, `widget` will be declared as a new function. does it make sense to move widget up to the top level of this module just not export it?